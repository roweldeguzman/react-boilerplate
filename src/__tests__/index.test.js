import React from "react";
import { render } from '@testing-library/react'

import Component from '../Application';

describe('App', () => {

  test("Renders correctly", () => {
    const { container } = render(<Component />)

    expect(container.firstChild).toMatchSnapshot();
  })

})