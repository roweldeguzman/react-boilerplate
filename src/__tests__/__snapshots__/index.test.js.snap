// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders correctly 1`] = `
<p>
  Application
</p>
`;
